routes/database.rb

```diff
@@ -0,0 +1,89 @@
+require 'sinatra/base'
+require_relative '../logging/logs'
+
+# Our app
+module Sinatra
+  # Our D Routing
+  module DBRoutes
+    # rubocop:disable LineLength, MethodLength, AbcSize, CyclomaticComplexity, PerceivedComplexity
+    def self.registered(app)
+      ##
+      # Database Resource
+      ##
+
+      app.get '/db/new/?' do
+        # get new database form
+        @projects = Project.all
+        erb :'database/create'
+      end
+
+      app.get '/db/:id/?' do
+        # view a database
+        @db = DBResource[id: params[:id]]
+        if @db.nil?
+          MyLog.log.fatal 'routes/database: Database not found'
```

Comment:
We might want to say "Database Resource not found" to avoid confusion with the app database